tests/test_parse_load_args.py

```diff
@@ -0,0 +1,160 @@
+import argparse
+import tempfile
+import uuid
+import textwrap
+from contextlib import contextmanager
+from os import chdir, getcwd
+from pathlib import Path
+from typing import List, Optional
+
+import mkosi
+import pytest
+from mkosi.backend import Distribution, MkosiArgs, MkosiException
+
+
+def parse(argv: Optional[List[str]] = None) -> MkosiArgs:
+    return mkosi.load_args(mkosi.parse_args(argv)["default"])
+
+@contextmanager
+def cd_temp_dir():
+    old_dir = getcwd()
+    tmp_dir = tempfile.TemporaryDirectory()
+
+    with tmp_dir:
+        chdir(tmp_dir.name)
+        try:
+            yield
+        finally:
+            chdir(old_dir)
+        tmp_dir.cleanup()
```

Comment:
When using with, you don't have to call `cleanup` anymore so this can be removed